packages/material-ui/src/Select/SelectInput.js

```diff
@@ -14,6 +14,10 @@ function areEqualValues(a, b) {
   return String(a) === String(b);
 }
 
+function isEmpty(display) {
+  return display == null || (typeof display === 'string' && !display.trim());
```

Comment:
```suggestion
  return !display || (typeof display === 'string' && !display.trim());
```

Reply:
0 should be accepted.

Reply:
Right, right, it also wouldn't work for strings like `'false'`. Disregard my suggestion please

Reply:
No problem, great to explore simplification opportunities :)